### PR TITLE
Syndicate Uplink Refresh: Reorganization, Rebalancing, Additions, and more! (WIP - DO NOT MERGE)

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

NOTE: STILL WORK IN PROGRESS. PR WAS MADE EARLY TO ADDRESS UNUSUAL ERRORS THAT ARE PREVENTING TESTING

Essentially a small rework of the syndicate uplink, this PR reorganizes the uplink, add some new gear for syndicate use, divides up some comboes, replaces the old syndicate bundles with new and improved bundles, tweaks flavor text, and rebalances various syndicate items.

## Why It's Good For The Game

The uplink is rather confusing to navigate, so this should hopefully make it more intuitive. All the new additions should open up new gameplay opportunities, and the weapon tweaks should give every weapon their own niche without as much overlap. 

## Changelog
:cl:
add: New items to the uplink! Including a double barrel shotgun (and shells), machine pistol, switchblades, door charges, night vision goggles, new ammo types, and more!
add: New bundles for traitors and nuclear operatives, great for beginners and vets alike! (WIP)
tweak: changes to weapon names, and descriptions. This serves to both add flavor and more importantly, tell players more about what they are buying.
balance: Changes the damage, ammo capacity, burst size, and more of various syndicate weapons. 
   *SAW: Default damage (45 -> 21), burst size (5 -> 8), ammo (50 -> 100). TC cost 18 -> 16.
   *M-90gl: Default damage (35 -> 30), burst size (3 -> 5), ammo (30 -> 50)
   *C-20r: ammo (24 -> 28)
   *Flechette: Default damage (15 -> 12), burst size (5 -> 8), ammo (40 -> 64)
balance: reduced beanbag stamina damage to 60 from 80.
balance: Various TC cost changes, most notably targeting syndie glue (2TC -> 4TC)
sound: replaces current traitor intro sound with the classic one because it sounds cooler.
fix: fixed some gramattical and spelling errors
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -142,15 +142,15 @@
...
+	desc = "An impossibly thin blade made of plastitanium, ending in a sharp diamond tip. It looks like it is capable of piercing through any amount of armor."
 	icon = 'icons/obj/items_and_weapons.dmi'
 	icon_state = "rapier"
 	item_state = "rapier"
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	force = 25
 	throwforce = 35
-	block_chance = 0
+	block_chance = 35
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/melee/misc.dm]: No, flat out no. the whole thing is that it dosnt have block, this isnt even done correctly and will block anything rather then just melee

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -766,7 +766,6 @@
 	item_state = "rsheath"
 	force = 5
 	throwforce = 15
-	block_chance = 30
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/belt.dm]: No this has the proc, that makes it melee only

File: code/game/objects/items/storage/boxes.dm [hunk 3]
```diff
@@ -771,6 +781,16 @@
 	for(var/i in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/techshell(src)
 
+/obj/item/storage/box/lasershot
+	name = "box of scatter laser shells"
+	desc = "A box full of scatter laser shells."
+	icon_state = "techslug_box"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/boxes.dm]: wrong icon, use lethal_box if anything

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 4]
```diff
@@ -84,7 +84,7 @@
 
 /obj/item/gun/ballistic/automatic/c20r
 	name = "\improper C-20r SMG"
-	desc = "A bullpup two-round burst .45 SMG, designated 'C-20r'. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
+	desc = "A bullpup two-round burst 10mm SMG, designated 'C-20r'. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/ballistic/automatic.dm:87]: this is still using .45s

File: code/modules/uplink/uplink_items.dm [hunk 5]
```diff
@@ -111,80 +111,41 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 //All bundles and telecrystals
 /datum/uplink_item/bundles_TC
-	category = "Bundles and Telecrystals"
+	category = "Bundles and Telecrystals."
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: Missing a medical bundle but meh

File: code/modules/uplink/uplink_items.dm [hunk 6]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+			For when you just need to spray and pray. Compatible with supressors."
+	item = /obj/item/gun/ballistic/automatic/pistol/machinepistol
+	cost = 10
 	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/dangerous/revolver
-	name = "Syndicate Revolver"
-	desc = "A brutally simple syndicate revolver that fires .357 Magnum rounds and has 7 chambers."
+/datum/uplink_item/munitions/revolver
+	name = "Nagant Revolver"
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: Wrong name, nagant revolvers are used in Russian mobs

File: code/modules/uplink/uplink_items.dm [hunk 7]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	 		Uses sniper ammo. \
-	 		Bullets tend to veer off-course. We are not responsible for any unintentional damage or injury resulting from inaacuracy."
+/datum/uplink_item/munitions/antitank
+	name = "Anti-Material Pistol"
+	desc = "Essentially an anti-material snipe rifle stripped down of all rifling. \
+			Fires .50 caliber rounds from a 5 round magazine, but with terrible accuracy. \
+			Cheaper than the revolver, and absolutely devastating if it hits"
 	item = /obj/item/gun/ballistic/automatic/pistol/antitank/syndicate
-	cost = 14
+	cost = 11
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: Er, this hardstun and delims... 14 is vault...

File: code/modules/uplink/uplink_items.dm [hunk 8]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	desc = "A box of shurikens and reinforced bolas from ancient Earth martial arts. They are highly effective \
-			 throwing weapons. The bolas can knock a target down and the shurikens will embed into limbs."
-	item = /obj/item/storage/box/syndie_kit/throwing_weapons
-	cost = 3
+/datum/uplink_item/munitions/doublebarrel
+	name = "Sawn-Off Double Barrel Shotgun"
+	desc = "A brutally simple double barrel shotgun, precut for portability. \
+			Designed for close quarter target elimination. Pre-loaded with buckshot shells"
+	item = /obj/item/gun/ballistic/revolver/doublebarrel/sawn
+	cost = 11
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: Why so much? It loads 2 shots...

File: code/modules/uplink/uplink_items.dm [hunk 9]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 	cost = 8
 	surplus = 40
 	include_modes = list(/datum/game_mode/nuclear)
 
-/datum/uplink_item/dangerous/smg
+/datum/uplink_item/munitions/smg
 	name = "C-20r Submachine Gun"
-	desc = "A fully-loaded Scarborough Arms bullpup submachine gun. The C-20r fires .45 rounds with a \
-			24-round magazine and is compatible with suppressors."
+	desc = "An ergonomic Scarborough Arms bullpup submachine gun that fires 10mm rounds in 2 round bursts from a 28-round magazine. \
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:309]: Also you still need to change the ammo to be correct

File: code/modules/uplink/uplink_items.dm [hunk 10]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	refund_path = /obj/item/guardiancreator/tech/choose/traitor
+	include_modes = list(/datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/dangerous/machinegun
+/datum/uplink_item/munitions/machinegun
 	name = "L6 Squad Automatic Weapon"
-	desc = "A fully-loaded Aussec Armoury belt-fed machine gun. \
-			This deadly weapon has a massive 50-round magazine of devastating 1.95x129mm ammunition."
+	desc = "A heavy duty Aussec Armoury belt-fed machine gun that carries a massive 100-round magazine of armor piercing 5.56�45mm ammunition that \
+			is fired in 8 round bursts. Requires two hands to fire. The L6 is ideal for suppressive fire and support roles.
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: CRITICAL Missing "  at the end

File: code/modules/uplink/uplink_items.dm [hunk 11]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	desc = "An additional 8-round slug magazine for use with the Bulldog shotgun. \
+			Ideal for engaging at longer ranges."
+	cost = 3
+	item = /obj/item/ammo_box/magazine/m12g/slug
+
 /datum/uplink_item/ammo/shotgun/dragon
 	name = "12g Dragon's Breath Drum"
 	desc = "An alternative 8-round dragon's breath magazine for use in the Bulldog shotgun. \
-			'I'm a fire starter, twisted fire starter!'"
+			Sets the target on fire, and generally gives them a bad time."
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: "Sends many small pestles that will set anyone on fire" or something more useful

File: code/modules/uplink/uplink_items.dm [hunk 12]
```diff
@@ -286,610 +247,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	include_modes = list(/datum/game_mode/nuclear)
-
-/datum/uplink_item/ammo/flechetteap
-	name = "Armor Piercing Flechette Magazine"
-	desc = "An additional 40-round flechette magazine; compatible with the Flechette Launcer. \
-			Loaded with armor piercing flechettes that very nearly ignore armor, but are not very effective agaisnt flesh."
-	item = /obj/item/ammo_box/magazine/flechette
-	cost = 2
-	include_modes = list(/datum/game_mode/nuclear)
+	cost = 5
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:743]: bit high...

File: code/modules/uplink/uplink_items.dm [hunk 13]
```diff
@@ -899,88 +755,248 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
+	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/dangerous/doublesword/get_discount()
+	return pick(4;0.8,2;0.65,1;0.5)
+
+/datum/uplink_item/cqc/rapier
+	name = " Plastitanium Rapier"
+	desc = "An incredibly sharp rapier capable of piercing all forms of armor with ease. \
+			The rapier comes with its own jet black sheath, however this will generally alert hostiles to your allegience. \
+			The rapier itself can be used to deflect melee strikes to some degree, and it can be used as a powerful projectile in a pinch.
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: CRITCAL missing a " at the end
Also FUCKEN FIX IT!
The Shielf is ment to fucken block not the rapier as well as only melee. Thats its downsides!!!

File: code/modules/uplink/uplink_items.dm [hunk 14]
```diff
@@ -1267,53 +1365,58 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
-	desc = "This red and black Syndicate space suit is less encumbering than Nanotrasen variants, \
-			fits inside bags, and has a weapon slot. Nanotrasen crew members are trained to report red space suit \
-			sightings, however."
+	desc = "This black and red Syndicate space suit is less encumbering than Nanotrasen variants, \
+			fitting inside bags, and being able to hold weapons. While fairly subtle in coloration, perceptive crew members will be quick to call it out."
 	item = /obj/item/storage/box/syndie_kit/space
 	cost = 4
+	exclude_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/suits/space_suit/nukeop
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: Useless their ship has hardsuits

File: code/modules/uplink/uplink_items.dm [hunk 15]
```diff
@@ -1324,7 +1427,12 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 			Nanotrasen crew who spot these suits are known to panic."
 	item = /obj/item/clothing/suit/space/hardsuit/syndi
 	cost = 8
-	exclude_modes = list(/datum/game_mode/nuclear) //you can't buy it in nuke, because the elite hardsuit costs the same while being better
+	exclude_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/suits/hardsuit/nukeop
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1427]: Useless they have hardsuits in the ship

File: code/modules/uplink/uplink_items.dm [hunk 16]
```diff
@@ -1335,47 +1443,179 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
+	name = "Combat Banana Shoes"
+	desc = "While making the wearer immune to most slipping attacks like regular combat clown shoes, these shoes \
+		can generate a large number of synthetic banana peels as the wearer walks, slipping up would-be pursuers. They also \
+		squeak significantly louder."
+	item = /obj/item/clothing/shoes/clown_shoes/banana_shoes/combat
+	cost = 6
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/tools/military_belt
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1489]: Restricted to nukies for a reason

File: code/modules/uplink/uplink_items.dm [hunk 17]
```diff
@@ -1335,47 +1443,179 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
+	surplus = 0
+
+/datum/uplink_item/tools/toolbox
+	name = "Full Syndicate Toolbox"
+	desc = "A particularly heavy and robust toolbox. It comes loaded with a full tool set including a \
+			multitool and combat gloves that are resistant to shocks and heat."
+	item = /obj/item/storage/toolbox/syndicate
+	cost = 1
+
+/datum/uplink_item/tools/surgerybag
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1510]: Adv surgery bag is missing please fix this

File: code/modules/uplink/uplink_items.dm [hunk 18]
```diff
@@ -1388,103 +1628,39 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
-/datum/uplink_item/device_tools/compressionkit
+/datum/uplink_item/devices/compressionkit
 	name = "Bluespace Compression Kit"
 	desc = "A modified version of a BSRPED that can be used to reduce the size of most items while retaining their original functions! \
 			Does not work on storage items. \
 			Recharge using bluespace crystals. \
 			Comes with 5 charges."
 	item = /obj/item/compressionkit
-	cost = 5
+	cost = 6
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: 💢 at lest 8 or something rawr

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Many fixes and such needs to be done
TLDR
Readd adv surgery kit for all gamemodes
Fix Rapiers to be as they were stat wise - desc changes are ok
Fix C-40 being called 10mm but still use .45s

File: code/modules/uplink/uplink_items.dm (hunk 15 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm:1427]: At highpop, they can be used to supply reinforcements. Not harmful to have em.

File: code/modules/uplink/uplink_items.dm (hunk 6 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm]: It's a flavor text. I changed the sprite too since a lot of people complained about the syndicate one.

File: code/modules/uplink/uplink_items.dm (hunk 5 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm]: WIP

File: code/modules/uplink/uplink_items.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm]: Nagant Revolver IS a revolver in game though that uses different ammo. naming it the same is confusion when they are different. 
Maybe call it `Classic Revolver`

File: code/modules/uplink/uplink_items.dm (hunk 15 above)
Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:1427]: they get like 7 suits tho. But I get ya point

File: modular_citadel/code/modules/projectiles/guns/ballistic/revolver.dm [hunk 19]
```diff
@@ -6,6 +6,6 @@
 	name = "\improper .357 revolver"
 	desc = "A typical revolver. Uses .357 ammo."
 
-/obj/item/gun/ballistic/revolver/syndie		//New and improved 100% syndicate technology!
+/obj/item/gun/ballistic/revolver/syndie		//Nice Looking
 	desc = "A suspicious revolver. Uses .357 ammo."
-	icon = 'modular_citadel/icons/obj/guns/revolver.dmi'
\ No newline at end of file
+	icon = 'icons/obj/guns/nagant.dmi'
```
Comment (Trilbyspaceclone) [inline modular_citadel/code/modules/projectiles/guns/ballistic/revolver.dm]: This is not done correctly...

File: code/modules/uplink/uplink_items.dm (hunk 12 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm:743]: Was 6 before. I might re-lower the RL price to 8 TC to compensate for expensive ammo though.

File: code/modules/uplink/uplink_items.dm (hunk 16 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm:1489]: Chest rig was never restricted to nukies. Grenadier belt was, but I have removed the nades from it, as a new grenadier kit will be made (for each mode) instead.

File: code/modules/uplink/uplink_items.dm (hunk 8 above)
Comment (Tupinambis) [reply code/modules/uplink/uplink_items.dm]: Lowered to 10

**Comment by lolman360** [comment]
does this port #9154 too?

**Comment by YakumoChen** [comment]
Door charges were removed from almost every means to access them (only a single bundle has it), because it's almost always a one-hit kill that you can't really plan for or see coming.

Reading this, almost all of it is pretty terrible and it's loaded with unwanted balance changes. I recommend you split up this into multiple PRs.

**Comment by Tupinambis** [comment]
> Door charges were removed from almost every means to access them (only a single bundle has it), because it's almost always a one-hit kill that you can't really plan for or see coming.
> 
> Reading this, almost all of it is pretty terrible and it's loaded with unwanted balance changes. I recommend you split up this into multiple PRs.

I would rather get direct feedback on the problem points, than be told "it's terrible".
On the note of door charges, at 4tc they are pretty steep, and you can effectively make a diet doorcharge by making an assembly with c4. They also don't do a lot of damage, and essentially behave like trap welderbombs.

**Comment by YakumoChen** [comment]
Airlock charge was removed in #1216 for the reasons predictably stated: it's basically an invisible instant kill. If you've never used one before, they use an explosion, which usually KOs or sets you on fire, while almost always creating a breach into space, it's almost invariably fatal due to the high damage and predictable after-effect damage.

-Adding a bunch of mundane guns or items you can get to the uplink doesn't really add much. It bogs the list down with garbage and do not serve as alternatives to existing options.
-The switchblade does not really compare well to the e-dagger, which is an overall superior item. 
-There are already several readily available shotguns on the station (and they are also craftable), shells can be VERY easily made with an autolathe.
-Night vision goggles are completely inferior to thermals, and are printable from every single protolathe once the research is done.
-The Uzi isn't really a good alternative to either the sketchkin or the revolver, which both already fit their respective niches.
**-Why the fuck are you randomly renaming all the guns and ammo?**
-"Rubber .357 shot" misses the point, it's a lethal revolver. That just makes it a fancy taser.
**-Undocumented nerf to sniper magazine size**
-**Don't randomly change the calibers of guns**
-**Why are you rebalancing things without giving reasons?**
-**ATOMIZE YOUR FUCKING PR**

**Comment by Tupinambis** [comment]
> Airlock charge was removed in #1216 for the reasons predictably stated: it's basically an invisible instant kill. If you've never used one before, they use an explosion, which usually KOs or sets you on fire, while almost always creating a breach into space, it's almost invariably fatal due to the high damage and predictable after-effect damage.
> 
> -Adding a bunch of mundane guns or items you can get to the uplink doesn't really add much. It bogs the list down with garbage and do not serve as alternatives to existing options.
> -The switchblade does not really compare well to the e-dagger, which is an overall superior item.
> -There are already several readily available shotguns on the station (and they are also craftable), shells can be VERY easily made with an autolathe.
> -Night vision goggles are completely inferior to thermals, and are printable from every single protolathe once the research is done.
> -The Uzi isn't really a good alternative to either the sketchkin or the revolver, which both already fit their respective niches.
> **-Why the fuck are you randomly renaming all the guns and ammo?**
> -"Rubber .357 shot" misses the point, it's a lethal revolver. That just makes it a fancy taser.
> **-Undocumented nerf to sniper magazine size**
> -**Don't randomly change the calibers of guns**
> -**Why are you rebalancing things without giving reasons?**
> -**ATOMIZE YOUR FUCKING PR**

A lot of it is flavor and such, Fully documented changes will come with the final release of this PR. I'll probably close this for now until it's 100% done.

File: code/modules/projectiles/projectile/bullets/revolver.dm [hunk 20]
```diff
@@ -27,6 +27,7 @@
 	name = ".357 bullet"
 	damage = 60
 
-/obj/item/projectile/bullet/a357/rubber
+/obj/item/projectile/bullet/a357rubber
 	name = ".357 rubber bullet"
-	stamina = 60
\ No newline at end of file
+	damage = 18
+	stamina = 70
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullets/revolver.dm:33]: Im really iffy about 70, maybe it be better if it was 60 to match normal harm

File: code/modules/uplink/uplink_items.dm [hunk 21]
```diff
@@ -500,19 +502,19 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A speed loader that contains seven additional lethal .357 Magnum rounds; compatible with the Nagant revolver. \
 			For when you really need a lot of things dead."
 	item = /obj/item/ammo_box/a357
-	cost = 2
+	cost = 1
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:505]: Wasnt this a high cost do to nukies getting it and using it for duel welding 120 harm with **tons** of ammo?

File: code/modules/uplink/uplink_items.dm [hunk 22]
```diff
@@ -615,14 +617,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	cost = 2
+	cost = 3
 
 /datum/uplink_item/ammo/smg/zzz
 	name = "10mm SMG Soporific Magazine"
 	desc = "An additional 28-round 10mm magazine suitable for use with the C-20r submachine gun. Loaded with soporific rounds that put the target to sleep. \
 			Remember that it takes about three shots to actually put the target to sleep, so plan accordinlgy."
 	item = /obj/item/ammo_box/magazine/smgm10mm/soporific
-	cost = 2
+	cost = 3
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:627]: Make this way higher, its a 3 hit sleep, thats pens all armor do to how sleep works

File: code/modules/uplink/uplink_items.dm [hunk 23]
```diff
@@ -672,6 +674,37 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+			Is incredibly effective at mowing down the unarmored masses of the crew."
+	item = /obj/item/ammo_box/magazine/mm556x45/hollow
+	cost = 5
+
+/datum/uplink_item/ammo/lmg/incen
+	name = "5.56x45mm (Incendiary) Box Magazine"
+	desc = "An alternative 75-round box magazine of incendiary rounds for the L6 Saw. \
+			Sets targets alight with every bullet, just be careful not to set your squad mates on fire."
+	item = /obj/item/ammo_box/magazine/mm556x45/incen
+	cost = 6
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:706]: Higher then it needs to be, Id suggest 5 and make AP 6 rather then incen

File: code/modules/uplink/uplink_items.dm [hunk 24]
```diff
@@ -842,7 +843,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 0
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
-datum/uplink_item/stealthy_weapons/taeclowndo_shoes
+datum/uplink_item/cqc/taeclowndo_shoes
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:846]: This is a bad pathing, likely broken if not messy

File: code/modules/uplink/uplink_items.dm [hunk 25]
```diff
@@ -1514,6 +1509,13 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 
 /datum/uplink_item/tools/surgerybag
 	name = "Syndicate Surgery Duffel Bag"
+	desc = "The Syndicate surgery duffel bag is a toolkit containing all surgery tools, surgical drapes, \
+			a Syndicate brand MMI, a straitjacket, a muzzle, and a full Syndicate Combat Medic Kit"
+	item = /obj/item/storage/backpack/duffelbag/syndie/surgery_adv
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1514]: These are mixed up a bit it seems

**Review by Trilbyspaceclone — COMMENTED** [review]
More broken/balance changes needed

File: code/modules/uplink/uplink_items.dm [hunk 26]
```diff
@@ -115,76 +115,37 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-		Includes a suit jacket that is padded with a robust liner."
-	item = /obj/item/storage/briefcase/modularbundle
-	cost = 12
+/datum/uplink_item/bundles_TC/toxic
+	name = "Toxic Bundle"
+	desc = "Toxins and toxin applicators, for chemical warfare: Contains a miniature energy crossbow, a sleepy pen, \
+			a dart gun, a poison kit, a box of spare syringes, and complementary syndicate cigarettes."
+	item = /obj/item/storage/box/syndie_kit/bundle/toxic
+	cost = 20 // normally 28
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:124]: You mean include for all these bundles CRITICAL

File: code/modules/uplink/uplink_items.dm [hunk 27]
```diff
@@ -286,610 +247,506 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-	item = /obj/item/book/granter/martial/cqc
-	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-	cost = 13
-	surplus = 0
+/datum/uplink_item/ammo/pistolap
+	name = "10mm Armour Piercing Magazine"
+	desc = "An additional 8-round 10mm magazine; compatible with the M92 Pistol. \
+			These rounds are less effective at injuring the target, but are useful for penetrating protective gear."
+	item = /obj/item/ammo_box/magazine/m10mm/ap
+	cost = 1
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:442]: You should make the better ammo cost more then basic

File: code/modules/uplink/uplink_items.dm [hunk 28]
```diff
@@ -286,610 +247,506 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
-			space a small item can."
-	item = /obj/item/gun/syringe/syndicate
-	cost = 4
-	surplus = 50
+/datum/uplink_item/ammo/pistolzzz
+	name = "10mm Soporific Magazine"
+	desc = "An additional 8-round 10mm magazine; compatible with the M92 Pistol. Loaded with soporific rounds that put the target to sleep. \
+			Remember that it takes about three shots to actually put the target to sleep, so plan accordinlgy."
+	item = /obj/item/ammo_box/magazine/m10mm/soporific
+	cost = 2
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:466]: Much more do to how easy it is to sleep people

File: code/modules/uplink/uplink_items.dm [hunk 29]
```diff
@@ -990,39 +1007,112 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 5
 	restricted = TRUE
 
+/datum/uplink_item/explosives/doorcharge
+	name = "Airlock Charge"
+	desc = "A small explosive charged that can be placed into the wiring of any airlock. \
+			The moment someone attempts to open the airlock, the charge will detonate, destroying the airlock and injuring anyone nearby."
+	item = /obj/item/doorCharge
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1014]: Please nerf this a bit to not use dev as its a sleep/auto kill for even armord targets, place 2 on sec and bam you killed off all of sec or a head

File: code/modules/uplink/uplink_items.dm [hunk 30]
```diff
@@ -990,39 +1007,112 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	restricted = TRUE
 
-/datum/uplink_item/explosives/grenadier
-	name = "Grenadier's belt"
-	desc = "A belt containing 26 lethally dangerous and destructive grenades. Comes with an extra multitool and screwdriver."
-	item = /obj/item/storage/belt/grenade/full
-	include_modes = list(/datum/game_mode/nuclear)
-	cost = 22
+/datum/uplink_item/explosives/tearstache
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1091]: Limit this to clownops

File: code/modules/uplink/uplink_items.dm [hunk 31]
```diff
@@ -1149,46 +1242,47 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
-		squeak significantly louder."
-	item = /obj/item/clothing/shoes/clown_shoes/banana_shoes/combat
-	cost = 6
-	surplus = 0
-	include_modes = list(/datum/game_mode/nuclear/clown_ops)
+/datum/uplink_item/stealthy_tools/jammer
+	name = "Radio Jammer"
+	desc = "This device will disrupt any nearby outgoing radio communication when activated. Does not affect binary chat."
+	item = /obj/item/jammer
+	cost = 5
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1285]: Can do with being 3 rather then 5 as its kinda obs and useless 9/10 times

File: code/modules/uplink/uplink_items.dm [hunk 32]
```diff
@@ -1335,50 +1438,189 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
+			They allow you to see organisms through walls by capturing the upper portion of the infrared light spectrum, \
+			emitted as heat and light by objects. Hotter objects, such as warm bodies, cybernetic organisms \
+			and artificial intelligence cores emit more of this light than cooler objects like walls and airlocks."
+	item = /obj/item/clothing/glasses/thermal/syndi
+	cost = 4
+
+/datum/uplink_item/tools/nightvision
+	name = "Night Vision Goggles"
+	desc = "Simple goggles that allow one to see in the dark. Incredibly useful for infiltration."
+	item = /obj/item/clothing/glasses/night
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:1467]: Make this prescription and cost 1

File: code/modules/uplink/uplink_items.dm [hunk 33]
```diff
@@ -1891,6 +2008,16 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	category = "(Pointless) Badassery"
 	surplus = 0
 
+/datum/uplink_item/badass/cxneb
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:2011]: Not pointless as thats the real thing

**Review by Trilbyspaceclone — COMMENTED** [review]
More fixes and such

File: code/modules/uplink/uplink_items.dm (hunk 24 above)
Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:846]: its aka missing the / at the start
